cypress/integration/InstancesTest.ts

```diff
@@ -55,7 +55,7 @@ describe("Instances Test", () => {
 
   describe("Instance Page - Instance six", () => {
     beforeEach(() => {
-      cy.visit("/instance/d7e13602-b046-4120-b377-15d61e21c31");
+      cy.visit("/instance/3543edaa-1851-4ad7-96be-ebde7d20d717");
```

Comment:
@vpellegrino the name of the test suite is "Instance six", I wanted to be sure that the details panel should contains different values than dummy data ("Instance one"). So feel free to check `d7e13602-b046-4120-b377-15d61e21c31a` or just rename the test suite `Instance Page - Instance one`.

Reply:
Ok, renaming the test suite, thanks @sjamboro